common/defaultRetrySettings.go

```diff
@@ -0,0 +1,11 @@
+package common
+
+// DefaultRetrySettings indicates what the "default" retry settings
+// are if it is not specified on an Activity or for any unset fields
+// if a policy is explicitly set on a workflow
+type DefaultRetrySettings struct {
+	InitialIntervalInSeconds   int32
+	MaximumIntervalCoefficient float64
```

Comment:
Why is it called `MaximumIntervalCoefficient`?  I think we should just call it `MaximumInterval` and use Duration as type.

Reply:
We needed to initialize maximum interval in the case maximum interval was not set by the user, but a default value might be less than the initial interval, which would fail validation.

Btw, this change is already part of code-complete (this is just moving code around) and Maxim and I walked through this specific choice to use Coefficient vs. Interval before we landed this. Happy to discuss offline if we want to make further changes here though.